tests/integration/ui_write/base.py

```diff
@@ -45,6 +47,22 @@ def container(self):
     def builder(self):
         return self.__builder
 
+    def assertDictEqual(self, d1, d2, msg=None):
+        """
+        Custom dictionary comparison routine treating NaN == NaN being true
+        """
+        self.assertIsInstance(d1, dict, 'First argument is not a dictionary')
+        self.assertIsInstance(d2, dict, 'Second argument is not a dictionary')
+
+        if d1 != d2 and len('\n'.join(difflib.context_diff(pprint.pformat(d1).splitlines(),
+            pprint.pformat(d2).splitlines()))) > 0:
+            standardMsg = '%s != %s' % _common_shorten_repr(d1, d2)
```

Comment:
Looks like you missed the function `_common_shorten_repr`  when copying over the assert method.

Reply:
@ajtritt Thanks, I'll fix that.